packages/components/src/loaders/Loader.tsx

```diff
@@ -0,0 +1,7 @@
+interface LoaderProps {
+  size?: string;
```

Comment:
Puedes limitar los valores como un enum de typescript, por ejemplo:

```suggestion
  size?: "sm" | "md" | "lg" | "xl";
```